connector/connect/server/src/test/scala/org/apache/spark/sql/connect/service/SparkConnectServiceE2ESuite.scala

```diff
@@ -77,10 +77,10 @@ class SparkConnectServiceE2ESuite extends SparkConnectServerTest {
 
       // query3 has not been submitted before, so it should now fail with SESSION_CLOSED
       // TODO(SPARK-46042) Reenable a `releaseSession` test case in SparkConnectServiceE2ESuite
```

Comment:
nit: this TODO is leftover after this PR

Reply:
Oops